src/CallHandler.js

```diff
@@ -187,6 +188,12 @@ function _onAction(payload) {
             );
         }
         else if (payload.type === 'screensharing') {
+            const screenCapErrorString = PlatformPeg.get().screenCaptureErrorString();
+            if (screenCapErrorString) {
+                _setCallState(undefined, newCall.roomId, "ended");
+                console.log("Can't capture screen: " + screenCapErrorString);
```

Comment:
uh, really? we just close the call and log to the console? Don't we need to pop up an error or something? I suppose https://github.com/vector-im/riot-web/issues/2504 meant that we were swallowing the error anyway, but this seems like a retrograde step.